index.html

```diff
@@ -1,269 +1,429 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
-    <link rel="stylesheet" href="./styles/style.css">
-    <link rel="stylesheet" href="./styles/reset.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <link rel="stylesheet" href="./styles/reset.css" />
     <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-DW1CGEB4Y2"></script>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-DW1CGEB4Y2"
+    ></script>
     <script>
       window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
 
-      gtag('config', 'G-DW1CGEB4Y2');
+      gtag("config", "G-DW1CGEB4Y2");
     </script>
     <title>Michael Oladele | Portfolio</title>
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <nav class="nav-bar">
-            <ul>
-                <li id="name"> <a href="#">Michael Oladele</a></li>
-                <li class="hamburger-icon"><a href="#"><i class="fas fa-bars"></i></a></li>
-                <li class="right-item"> <a href="#">Portfolio</a></li>
-                <li class="right-item"> <a href="#">About</a></li>
-                <li class="right-item"> <a href="#">Contact</a></li>
-                <li class="right-item"> <a href="#"><i class="fas fa-envelope-square"></i></a></li>
-            </ul>
-        </nav>
+      <nav class="nav-bar">
+        <ul>
+          <li id="name"><a href="#">Michael Oladele</a></li>
+          <li class="hamburger-icon">
+            <a href="#"><i class="fas fa-bars"></i></a>
+          </li>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close buttons on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.